ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/review/v1/review/_cq_template.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="Review"
```

Comment:
We can keep the fieldType in JCR for consistency, but the code should fallback even if fieldType is not present in JCR

Reply:
Done